browser/brave_wallet/brave_wallet_tab_helper_browsertest.cc

```diff
@@ -22,23 +23,82 @@
 #include "chrome/test/base/ui_test_utils.h"
 #include "components/network_session_configurator/common/network_switches.h"
 #include "components/permissions/fake_usb_chooser_controller.h"
+#include "components/sessions/content/session_tab_helper.h"
 #include "content/public/browser/browser_task_traits.h"
 #include "content/public/test/browser_test.h"
 #include "content/public/test/browser_test_utils.h"
+#include "content/public/test/test_navigation_observer.h"
 #include "content/public/test/web_contents_tester.h"
 #include "net/dns/mock_host_resolver.h"
 
 namespace {
 
 const char kEmbeddedTestServerDirectory[] = "brave-wallet";
 
+content::WebContents* GetWebContentsFromTabId(
```

Comment:
use the one from common?

Reply:
actually definitely use the one from common or convert this one to not use the extensions method

Reply:
done